packages/pf-codemods/index.js

```diff
@@ -43,8 +44,15 @@ function printResults(engine, results, format) {
 
 function main() {
   let currentOptions;
+
   try {
-    currentOptions = options.parse(process.argv);
+    let arguments = lodash.cloneDeep(process.argv);
+
+    // remove --rules for parsing purposes
+    if (process.argv.includes('--rules')) {
+      arguments.splice(arguments.indexOf('--rules'),2);
+    }
+    currentOptions = options.parse(arguments);
   } catch (error) {
     console.error(error.message);
     return 2;
```

Comment:
Line 62-83 is what needs editing. `rules: undefined` needs to be populated and the `.eslintrc.json` might need to be modified to not use the recommended rules from the plugin (aka ALL rules).

Reply:
Done